articles/multifactor-authentication/duo/user-guide.md

```diff
@@ -0,0 +1,40 @@
+# How to Use Duo
```

Comment:
I wasn't sure how to handle this, this page is meant for our customer's users so they might not know anything about Auth0 or Lock.